src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,79 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+type PropsPagination = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}: PropsPagination) => {
+  const items = getNumbers(1, total);
```

Comment:
Creating an array of items from 1 to `total` seems unnecessary. Since you only need the total number of pages, you can calculate `totalPages` directly using `total` and `perPage` without generating an intermediate array.

Reply:
Ok